docs/protocol/extension_sequence.md

```diff
@@ -35,24 +35,26 @@ supported, Triton reports “sequence” in the extensions field of its
 Server Metadata.
```

Comment:
You should add sentences about "sequence(string_id)" extension and note what it adds. Similar to what is done in the model_repository extension. https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_model_repository.md#model-repository-extension

Reply:
Added